packages/hub/config/staging.json

```diff
@@ -5,6 +5,8 @@
     "allowedChannels": "898941895525425172"
   },
   "cardDrop": {
+    "sku": "0x18367eee0b96d5f6e46e172ee78585d22f464884e27e7e843031c03a05af16d2",
+    "FIXME": "remove the above?",
```

Comment:
I’m unsure whether the SKU will differ between testnet and mainnet but will follow up on this once I know.